documents/plan.md

```diff
@@ -1,11 +1,16 @@
 # Project plan
 
 ## Language evaluation
+I'll know that my language has accomplished its goals when it is possible to get from it any output that is even possible using the back end I have in mind. It should also take less time to take already-existing tab and re-write it in this language than it takes to re-write it in a graphical tab editor. Ideally, it should require fewer characters in this language to invoke certain behaviors (e.g. tripleted notes) than the back end requires.
```

Comment:
"I'll know that my language has accomplished its goals when it is possible to get from it any output that is even possible using the back end I have in mind." ← I'm not sure what this means.